packages/pds/src/db/tables/user-account.ts

```diff
@@ -8,6 +8,7 @@ export interface UserAccount {
   passwordResetToken: string | null
   passwordResetGrantedAt: string | null
   invitesDisabled: Generated<0 | 1>
+  inviteNote: string | null
```

Comment:
this tables starting to get pretty bloated/multi-purpose, may be worth splitting it up soon

probably doesn't need to land in this PR, altho I'm a bit worried about adding in an unconstrained text column to a table that is in pretty low latency queries (like search). Probably worth putting a max length on the note field

Reply:
agreed. I wasn't sure if adding it here was wise but it made sense to keep this along with `invitesDisabled` column.